Test/ItemTests.cs

```diff
@@ -1,10 +1,17 @@
-﻿using TMS_Selenium.Utils;
+﻿using Allure.Net.Commons;
+using Allure.NUnit.Attributes;
+using TMS_Selenium.Utils;
 
 namespace TMS_Selenium.Test
 {
+    [AllureFeature("Shopping")]
     public class ItemTests : BaseTest
     {
         [Test]
+        [AllureDescription("Test to check information on the Item page: the price and name of the product are the same as on the Products page, " +
+            "and the \"Add to cart\" button is displayed")]

```

Comment:
Also you can put Add to cart in single quotes: "... and the 'Add to cart' button.."